src/components/SearchAndFilter/SearchAndFilter.tsx

```diff
@@ -25,12 +25,17 @@ export type Props = {
    * A function that is called when the search data changes.
    */
   returnSearchData: (searchData: SearchAndFilterChip[]) => void;
+  /**
+   * A function that is called when the height of the search container changes.
+   */
+  onHeightChange?: () => void;
```

Comment:
Maybe better `onSearchBoxExpand`?

Reply:
I don't think that name makes it clear that the function is going to be called also when the search box is collapsed, not only when it's expanded.